ArduPlane/quadplane.cpp

```diff
@@ -730,10 +738,10 @@ bool QuadPlane::setup(void)
     }
     AP_Param::load_object_from_eeprom(loiter_nav, loiter_nav->var_info);
 
-    motors->init((AP_Motors::motor_frame_class)frame_class.get(), (AP_Motors::motor_frame_type)frame_type.get());
+    motors->initialise(motor_class, motor_type);
 
     if (!motors->initialised_ok()) {
-        AP_BoardConfig::config_error("unknown Q_FRAME_TYPE %u", (unsigned)frame_type.get());
+        AP_BoardConfig::config_error("init failed: class: %u type: %u", motor_class, motor_type);
```

Comment:
We do appear to have a valid motors object here, so shouldn't at least one of the string-getting methods work here?!

Reply:
I was thinking the class, type or both might be out of range of the respective enum, and in that case the integer param value would be more appropriate to report.

Reply:
initialized_ok is a boolean, it doesn't know why the failure occurred